src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/primary/index.js

```diff
@@ -367,7 +372,9 @@ class GmailAppSidebarPrimary {
         buttonContainer = existingButtonContainer;
       } else {
         buttonContainer = document.createElement('div');
-        buttonContainer.className = idMap('sidebar_button_container');
+        buttonContainer.className = cx(
+          [legacyIdMap, idMap].map((f) => f('sidebar_button_container'))
+        );
```

Comment:
only need idMap for "sidebar_button_container"

Reply:
e38cc028